agir/activity/components/page__activities/ActivityCard.js

```diff
@@ -4,7 +4,7 @@ import React, { useMemo } from "react";
 import styled from "styled-components";
 
 import { activityStatus } from "@agir/activity/common/helpers";
-import { dateFromISOString, displayHumanDate } from "@agir/lib/utils/time";
+import { dateFromISOString, displayHumanDate, displayHumanDay } from "@agir/lib/utils/time";
```

Comment:
```suggestion
import { dateFromISOString, displayHumanDate } from "@agir/lib/utils/time";
```
Plus besoin d'importer cette fonction